pom.xml

```diff
@@ -12,13 +12,15 @@
 
 	<properties>
 		<!-- OUR VERSION -->
-		<revision>1.0-SNAPSHOT</revision>
+		<revision>1.0.0-SNAPSHOT</revision>
 
 		<!-- DEPENDENCIES VERSION -->
 		<jdk.version>1.8</jdk.version>
-		<cds.services.version>1.7.0</cds.services.version>
-		<spring.boot.version>2.3.1.RELEASE</spring.boot.version>
-
+		<cds.services.version>1.8.1</cds.services.version>
+		<spring.boot.version>2.3.2.RELEASE</spring.boot.version>
+		<cds4j.version>1.10.0</cds4j.version>
+		<spring.boot.version>2.2.6.RELEASE</spring.boot.version>
+		<ngdbc.version>2.4.76</ngdbc.version>
```

Comment:
JDBC driver version is managed through our `cds-feature-hana`. We should remove the below profiles section as suggested in my other comment and then we also can rely on `cds-feature-hana` managing the JDBC driver for us.